### DISCUSSION TITLE
Is there have the source code of this demo : https://templates.tiptap.dev/cHcFhqC9R7

### DISCUSSION THREAD

**Discussion by u9646** [ueberdosis/tiptap#4979] [General Questions] [answered]
hi，where can I find the source code of this example, thank you

https://templates.tiptap.dev/cHcFhqC9R7

**Comment by schnubor** [comment]
Yes, it's here: https://github.com/ueberdosis/tiptap-templates

**Reply by u9646** [reply]
thanks for your reply，but the link it seem has 404?

**Reply by schnubor** [reply]
The repo is private, you might need to signup for TipTap in order to get access.

**Reply by u9646** [reply]
Should I register here： https://cloud.tiptap.dev/register ？

**Reply by catelina0805** [reply]
Have you downloaded the source code?

**Reply by nperez0111** [reply, marked as answer]
To access, you must create a free tiptap account & you can then access it from here: https://cloud.tiptap.dev/react-templates

Clicking this link will go to the sign in page if not already signed in, you can create an account from here too

**Reply by abtin4079** [reply]
Hi, I am already signed in to Tiptap, and still, I can not access the GitHub page. What should I do?

**Reply by rubanraj54** [reply]
Same here, I have signed up for an account but cannot access the repo. @nperez0111 @schnubor
